conn.go

```diff
@@ -356,12 +356,6 @@ func (c *boltConn) Close() error {
 		return nil
 	}
 
-	if c.transaction != nil {
```

Comment:
why are we no longer rolling back the transaction?

Reply:
This was a duplicate condition.

Reply:
from #29

Reply:
cool.